tests/api/v2/test_knowledge.py

```diff
@@ -107,6 +127,38 @@ async def test_display_relationships(knowledge_webapp, aiohttp_client):
     assert response[0]['source']['source'] == 'this_is_a_valid_operation_id'
 
 
+async def test_display_operation_relationships(knowledge_webapp, aiohttp_client):
+    client = await aiohttp_client(knowledge_webapp)
+    op_id_test = 'this_is_a_valid_operation_id'
+    fact_data_a = {
+        'trait': 'a',
+        'value': '1',
+        'source': op_id_test
+    }
+    fact_data_b = {
+        'trait': 'b',
+        'value': '2',
+        'source': op_id_test
+    }
+    relationship_data = {
+        'source': fact_data_a,
+        'edge': 'gamma',
+        'target': fact_data_b,
+        'origin': op_id_test
+    }
+    await client.post('/relationships', json=relationship_data, headers=headers)
+    resp = await client.get(f'/relationships/{op_id_test}', headers=headers)
+    data = await resp.json()
+    response = data['found']
+
+    assert len(response) == 1
+    assert response[0]['source']['trait'] == 'a'
+    assert response[0]['source']['value'] == '1'
+    assert response[0]['edge'] == 'gamma'
+    assert response[0]['origin'] == 'this_is_a_valid_operation_id'
+    assert response[0]['source']['source'] == 'this_is_a_valid_operation_id'
```

Comment:
can we also verify the target?

Reply:
should be updated now to check for target